patches/openwrt/0007-net-mac80211-override-incompatible-basic-rates-for-mesh.patch

```diff
@@ -35,24 +50,15 @@ index 0000000000000000000000000000000000000000..19ca64d5012d2974ab2a48e9363ecaa3
 + net/mac80211/mesh.c | 7 +++++--
 + 1 file changed, 5 insertions(+), 2 deletions(-)
 +
-+diff --git a/net/mac80211/mesh.c b/net/mac80211/mesh.c
-+index 25223184d6e5..53b5339be5d0 100644
 +--- a/net/mac80211/mesh.c
 ++++ b/net/mac80211/mesh.c
-+@@ -92,8 +92,11 @@ bool mesh_matches_local(struct ieee80211_sub_if_data *sdata,
++@@ -94,9 +94,6 @@ bool mesh_matches_local(struct ieee80211
 + 	ieee80211_sta_get_rates(sdata, ie, sband->band,
 + 				&basic_rates);
 + 
 +-	if (sdata->vif.bss_conf.basic_rates != basic_rates)
 +-		return false;
-++	if (sdata->vif.bss_conf.basic_rates != basic_rates) {
-++		wiphy_warn(sdata->wdev.wiphy,
```

Comment:
Would it help to use `wiphy_warn_ratelimited` instead?
See https://github.com/torvalds/linux/blob/v6.6/include/net/cfg80211.h#L9094-L9097